packages/blade/.babelrc.js

```diff
@@ -0,0 +1,93 @@
+module.exports = {
+  env: {
+    development: {
+      presets: ['@babel/preset-typescript', 'module:metro-react-native-babel-preset'],
+      plugins: [
+        [
+          'module-resolver',
+          {
+            alias: {
+              '@storybook/react': '@storybook/react-native',
+            },
+          },
+        ],
+        [
+          '@babel/plugin-transform-react-jsx',
```

Comment:
added this to support the new transform from react and hance we can get rid of `import React from 'react'` statements.

This is explicitly required for react-native projects.For react projects `@babel/preset-react` already adds this

Reply:
I thought react 17 supports this without any extra config

Reply:
I assumed the same but then was bit by errors when the writing tests 😄